bytestring.cabal

```diff
@@ -124,24 +125,27 @@ library
                      NamedFieldPuns
 
   ghc-options:      -Wall -fwarn-tabs -Wincomplete-uni-patterns
+                    -optP -Wall -optP -Werror=undef
```

Comment:
`-optP -Werror=undef` instead of `-Wcpp-undef` is another sacrifice to ghc-8.0 compatibility.